specification/ai/Face/models.common.tsp

```diff
@@ -0,0 +1,196 @@
+  url: url;
+}
+
+@doc("Model for add face result.")
+model AddFaceResult {
+  @doc("The persisted face id.")
+  persistedFaceId: string;
+}
+
+@doc("Query parameters for whether response resource should include recognition model.")
+model ReturnRecognitionModelOptions {
+  @query
+  @doc("Return 'recognitionModel' or not. The default value is false.")
+  returnRecognitionModel?: boolean;
+}
+
+@doc("Training status of a container")
+model TrainingStatus {
+  @doc("Training status of the container.")
+  @lroStatus
+  status: TrainingStatusType;
+
+  @doc("A combined UTC date and time string that describes the created time of the person group, large person group or large face list.")
+  createdDateTime: utcDateTime;
+
+  @doc("A combined UTC date and time string that describes the last modify time of the person group, large person group or large face list, could be null value when the group is not successfully trained.")
+  lastActionDateTime: utcDateTime;
+
+  @doc("A combined UTC date and time string that describes the last successful training time of the person group, large person group or large face list.")
+  lastSuccessfulTrainingDateTime: utcDateTime;
+
+  @doc("Show failure message when training failed (omitted when training succeed).")
+  message?: string;
+}
+
+@doc("Type of training status.")
+enum TrainingStatusType {
+  @doc("If the training process is waiting to perform, the status is notstarted.")
+  nonstarted,
+
+  @doc("If the training is ongoing, the status is running.")
+  running,
+
+  @lroSucceeded
+  @doc("Status succeed means this person group or large person group is ready for Face - Identify, or this large face list is ready for Face - Find Similar.")
+  succeeded,
+
+  @lroFailed
+  @doc("Status failed is often caused by no person or no persisted face exist in the person group or large person group, or no persisted face exist in the large face list.")
+  failed,
+}
+
+@doc("Identify result.")
+model IdentifyResult {
+  @doc("Face ID of the query face.")
+  faceId: string;
+
+  @doc("The top candidate returned from the database.")
+  candidates: IdentifyCandidate[];
+}
+
+@doc("Candidate for identify call.")
+model IdentifyCandidate {
+  @doc("PersonId of the candidate.")
+  personId: string;
+
+  @doc("Confidence value of the candidate.")
+  confidence: float32;
+}
+
+@doc("Verify result.")
+model VerifyResult {
+  @doc("True if the two faces belong to the same person or the face belongs to the person, otherwise false.")
+  isIdentical: boolean;
+
+  @doc("Confidence score of whether the two faces belong to the same person or whether the face belongs to the person. The range is [0, 1].")
+  confidence: float32;
+}
+
+@doc("Similar face searching mode.")
+enum FindSimilarMatchMode {
+  @doc("Match person.")
+  matchPerson,
+
+  @doc("Match face.")
+  matchFace,
+}
+
+@doc("Response body for find similar face operation.")
+model FindSimilarResult {
+  @doc("Confidence value of the candidate.")
```

Comment:
As with other cognitive services, I recommend documenting this is between 0.0 and 1.0.